<?xml version="1.0"?>
<launch>
    <node pkg="tf" type="static_transform_publisher" name="ptz_link_to_camera" args="0.0 0.0 0.0 0.0 0.0 0.0 /ptz_link /camera 10" />
    
    <node name="camera1394_node" pkg="camera1394" type="camera1394_node" output="screen" respawn="true">
        <param name="guid" value="000a470104055023" />
        <param name="reset_on_open" value="true" />
        
        <param name="video_mode" value="640x480_yuv422" />
        
        <param name="frame_rate" value="30.0" />
        
        <param name="auto_whitebalance" value="4" /> <!-- one push -->

        <param name="auto_brightness" value="3" />
        <param name="brightness" value="0" />

        <param name="auto_gain" value="3" />
        <param name="gain" value="0" />

        <param name="auto_shutter" value="3" />
        <param name="shutter" value="400" />
    </node>

    <node name="config_loader" pkg="dynamic_reconfigure" type="dynparam" args="load /camera1394_node $(find robot_detection)/launch/marlin.yaml -t 5" />
</launch>
